.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

```suggestion
blank_issues_enabled: true
```
I'd prefer enabling blank issues since these may come in handy in situations where it is neither a bug nor a feature.